core/api/src/services/ledger/admin-legacy.ts

```diff
@@ -54,7 +56,7 @@ export const updateLndEscrow = async (amount: Satoshis) => {
     return { ...escrowData, updated: false }
   }
 
-  const entry = MainBook.entry("escrow")
+  const entry = MainBookAdmin.entry("escrow")
```

Comment:
I wonder if we should keep `MainBook` for writes and use `MainBookAdmin` just for queries 🤔

Reply:
initially I had `MainBookReadOnly` but even balance needs write permissions to save snapshot.. I will merge it to validate and if it is the case I will update it in the next PR